### PR TITLE
Route models

### PR DESCRIPTION
**Pull request by alickbass**
This pull request contains all the models, mentioned [here](https://github.com/toehook/slap-ios/issues/1)

### CODE REVIEW

File: Slap/Grade.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+//  Created by Oleksii on 08/02/2017.
+//  Copyright © 2017 WeAreReasonablePeople. All rights reserved.
+//
+
+import Foundation
+
+extension Route {
+    struct Grade {
+        enum Letter: Character {
+            case a = "A", b = "B", c = "C"
```
Comment (davidvanleeuwen) [inline Slap/Grade.swift]: Maybe it should be a bit more generic, as the US uses a different grading system: https://en.wikipedia.org/wiki/Grade_(bouldering)

File: Slap/Builder.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
+//
+//  Builder.swift
+//  Slap
+//
+//  Created by Oleksii on 08/02/2017.
+//  Copyright © 2017 WeAreReasonablePeople. All rights reserved.
```
Comment (davidvanleeuwen) [inline Slap/Builder.swift]: Wait what? 🤣

File: Slap/Grade.swift (hunk 1 above)
Comment (alickbass) [inline Slap/Grade.swift]: As I mentioned in the Issue, I think it's better to start with European grading, as there is no good way to convert between different grading systems

File: Slap/Builder.swift (hunk 2 above)
Comment (alickbass) [inline Slap/Builder.swift]: oops)))) sorry, will fix this one

File: Slap/Grade.swift (hunk 1 above)
Comment (alickbass) [inline Slap/Grade.swift]: According to [this page](https://www.mec.ca/en/explore/climbing-grade-conversion/) we can try to convert between different grade systems, but I think that it should be done is separate PR and should be heavily tested
